apiserver/params/internal.go

```diff
@@ -297,17 +297,34 @@ type RelationUnitsSettings struct {
 // RelationResult returns information about a single relation,
 // or an error.
 type RelationResult struct {
+	Error            *Error                `json:"error,omitempty"`
+	Life             Life                  `json:"life"`
+	Id               int                   `json:"id"`
+	Key              string                `json:"key"`
+	Endpoint         multiwatcher.Endpoint `json:"endpoint"`
+	OtherApplication string                `json:"other-application,omitempty"`
+}
+
+// RelationResults holds the result of an API call that returns
+// information about multiple relations.
+type RelationResults struct {
```

Comment:
Maybe move this above RelationResult so that all the RelationResult structs are next to each other?